great-idea/css/index.css

```diff
@@ -79,11 +79,13 @@ nav a{
     text-decoration: none;
      color: aqua;
 }
-
+.demand-main{
+    display: flex;
+    justify-content: space-between;
+}
 .demand{
-    display: inline-block;
+    display: flex
```

Comment:
Don't forget the `;`. The properties don't get applied if syntax is wrong.

In this case, if you put back the `;`, what happens to the `h1` elements and the button?